runtime/core/exec_aten/util/tensor_util.h

```diff
@@ -338,14 +338,19 @@
  *
  * @param[in] cond the condition to check
  */
-#define ET_LOG_AND_RETURN_IF_FALSE(cond)           \
+#define ET_LOG_AND_RETURN_UNLESS(cond)             \
```

Comment:
These feel like they should live near https://github.com/pytorch/executorch/blob/d99970ba72d44ab80087fc95bfdf68f3d363f87d/runtime/core/error.h#L121 and be called ET_CHECK_OR_RETURN_FALSE to be consistent with the core helpers. But that's a separate issue since this PR doesn't change where they live.